lms/templates/register-form.html

```diff
@@ -234,7 +229,7 @@ <h2 class="sr">${_("Additional Personal Information")}</h2>
     % endif
   </ol>
 </div>
-<%static:optional_include_mako file="register_extra.html" is_theming_enabled="True" />
```

Comment:
Wait, aren't we still using this for the research paragraph?

Reply:
not on the old register_form, this is the non-logistration...